Resources/Computing/BatchSystems/SLURM.py

```diff
@@ -43,7 +43,8 @@ def submitJob(self, **kwargs):
     queue = kwargs['Queue']
     submitOptions = kwargs['SubmitOptions']
     executable = kwargs['Executable']
-    numberOfProcessors = kwargs['NumberOfProcessors']
+    numberOfProcessors = kwargs.get('NumberOfProcessors', '1')
+    numberOfNodes = kwargs.get('NumberOfNodes', '1')
```

Comment:
I would do:

```suggestion
    numberOfProcessors = kwargs.get('NumberOfProcessors', 1)
    numberOfNodes = kwargs.get('NumberOfNodes', 1)
```

and then few lines below

```
      cmd += "--nodes=%d " % numberOfNodes
      cmd += "--cpus-per-task=%d " % numberOfProcessors
```

As is seems more tidy to me. 
Same for SSHComputingElement.py

Reply:
Yes you're right for `NumberOfProcessors`, but I would like to keep `NumberOfNodes` as a string. 
Indeed,  Slurm for instance is able to allocate a certain number of nodes between a min and a max value.
Thus, I would like to be able to set values such as: `2-5` that would correspond to `<minNode>-<maxNode>`.

Reply:
Fine, but please in the code explain why a string is needed.